src/phoenix/db/migrations/versions/bc8fea3c2bc8_add_prompt_tables.py

```diff
@@ -0,0 +1,184 @@
+"""Add prompt tables
+
+Revision ID: bc8fea3c2bc8
+Revises: 4ded9e43755f
+Create Date: 2024-12-16 15:45:01.090563
+
+"""
+
+from typing import Any, Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy import JSON
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.ext.compiler import compiles
+
+
+class JSONB(JSON):
+    # See https://docs.sqlalchemy.org/en/20/core/custom_types.html
+    __visit_name__ = "JSONB"
+
+
+@compiles(JSONB, "sqlite")
+def _(*args: Any, **kwargs: Any) -> str:
+    # See https://docs.sqlalchemy.org/en/20/core/custom_types.html
+    return "JSONB"
+
+
+JSON_ = (
+    JSON()
+    .with_variant(
+        postgresql.JSONB(),  # type: ignore
+        "postgresql",
+    )
+    .with_variant(
+        JSONB(),
+        "sqlite",
+    )
+)
+
+revision: str = "bc8fea3c2bc8"
+down_revision: Union[str, None] = "4ded9e43755f"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "prompt_tag_configs",
+        sa.Column("id", sa.Integer, primary_key=True),
+        sa.Column("name", sa.String, nullable=False, unique=True),
+        sa.Column("description", sa.String),
+        sa.UniqueConstraint("name", name="uq_prompt_tag_configs_name"),
+    )
+
+    op.create_table(
+        "prompts",
+        sa.Column("id", sa.Integer, primary_key=True),
+        sa.Column("name", sa.String, unique=True, nullable=False),
+        sa.Column("description", sa.String),
```

Comment:
Just a note to future us - we probably need to account for if the prompt is "cloned"